### PR TITLE
Agile Practices: Modernization

### PR DESCRIPTION
**Pull request by joemoore**
This is the first of hopefully many submissions to a new section of the Dev Center: Agile Practices (final name TBD). 

These are being migrated from an internal VMware site to the Dev Center in stages. 

This PR contains the following: 
- Landing page for Agile Practices at `/practices`. Note that this is **not** linked to from any menu yet.
- Three "Modernization" practices: Swift Method, Event Storming, and Boris

The intention is to soft-release this section and get a bit of feedback by sharing a direct link. After incorporating the feedback in future PRs we can add it to the site's nav.

### CODE REVIEW

File: themes/docsy/layouts/practices/single.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
+{{ define "main" }}
+{{ partial "amplitude.html" (dict "context" . "event" "sample viewed") }}
```
Comment (bryanfriedman) [inline themes/docsy/layouts/practices/single.html:2]: Let's remove this for now, or change the event type to "practice viewed" or something. This is metrics tracking for the Amplitude tool.

File: themes/docsy/layouts/practices/single.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <div class="github-imported-sample col-9 steps">
+            {{ if .Params.readme }}
+            {{ $user_repo := (urls.Parse .Params.repo).Path }}
+            {{ $resp := getJSON (print "https://api.github.com/repos" $user_repo "/readme") }}
+            <!-- RegEx replace removes any image references which could be broken -->
+            {{ $resp.content | base64Decode | replaceRE "\\!\\[(.*)\\]\\((.*)\\)" "IMAGE" | markdownify }}
+            {{ else }}
+            {{ .Content }}
+            {{ end }}
+        </div>
```
Comment (bryanfriedman) [inline themes/docsy/layouts/practices/single.html:109]: These lines can be removed - they are specific to the "samples" pages for when there is a readme to import.

**Review by bryanfriedman — CHANGES_REQUESTED** [review]

File: themes/docsy/layouts/practices/single.html (hunk 1 above)
Comment (joemoore) [reply themes/docsy/layouts/practices/single.html]: Removed amplitude but kept the main wrapper on line 1 and end of file.

File: themes/docsy/layouts/practices/single.html (hunk 2 above)
Comment (joemoore) [reply themes/docsy/layouts/practices/single.html]: Removed github class and readme stuff; kept `{{ .Content }}`

**Comment by BrianMMcClain** [comment]
@joemoore My only suggestion would be to move each of the individual practices from `content.md` to `index.md` and updating all of links referencing the, this will make the links `/developer/practices/boris` instead of `/developer/practices/boris/content` for example. Otherwise looks great!

**Comment by joemoore** [comment]
Great idea on paths. Done

**Comment by bryanfriedman** [comment]
@joemoore Nice, yeah index.md is the way to go. 👍  For more context on _index.md vs. index.md vs. <name>.md: https://gohugo.io/content-management/organization/

**Review by BrianMMcClain — APPROVED** [review]

**Review by bryanfriedman — APPROVED** [review]

**Pull request merged**